core/src/main/java/de/jplag/reporting/reportobject/ReportObjectFactory.java

```diff
@@ -169,7 +171,7 @@ private void writeOverview(JPlagResult result, String path) {
         int shownComparisons = totalComparisons > numberOfMaximumComparisons ? numberOfMaximumComparisons : totalComparisons;
         int missingComparisons = totalComparisons > numberOfMaximumComparisons ? (totalComparisons - numberOfMaximumComparisons) : 0;
         logger.info("Total Comparisons: {}. Shown Comparisons: {}. Missing Comparisons: {}.", totalComparisons, shownComparisons, missingComparisons);
```

Comment:
This line was not part of the PR, but could you change it to:
```suggestion
        logger.info("Total Comparisons: {}. Shown Comparisons: {}. Omitted Comparisons: {}.", totalComparisons, shownComparisons, missingComparisons);
```